books/free-programming-books-langs.md

```diff
@@ -403,6 +404,14 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Object-Oriented Programming in the BETA Programming Language](https://beta.cs.au.dk/Books/) - Ole Lehrmann Madsen, Birger Møller-Pedersen, Kristen Nygaard
 
 
+### Blockchain
+
+* [Blockchain for Dummies](https://www.ramonmillan.com/documentos/bibliografia/BlockchainForDummies_Wiley.pdf) - Tiana Laurence (PDF)
```

Comment:
This appears not to be an authorized distribution. Please see CONTRIBUTING.md guidelines.